calculatorApi1.js

```diff
@@ -0,0 +1,48 @@
+const express = require("express");
+const cors = require("cors");
+const app = express();
+app.use(cors())
+const port = 3000
+
+const add = (arg1, arg2) =>{
+    return parseInt(arg1) + parseInt(arg2)
+}
+
+const sub = (arg1,arg2) => {
+    return parseInt(arg1) - parseInt(arg2)
+}
```

Comment:
In general I would do my parseInt before sending arguments into the function. Or running checks into the function. Writing a sum* function that expects two numbers as strings is a bit counter intuitive.